jwql/tests/test_permissions.py

```diff
@@ -51,7 +51,7 @@ def test_directory(test_dir=TEST_DIRECTORY):
 
     yield test_dir
     if os.path.isdir(test_dir):
-        os.remove(test_dir)
```

Comment:
Tests are passing locally. When testing this on one of the servers, it would be helpful to run the tests there as well, in addition to making sure that file/directory permissions end up being what we expect.